src/components/home/Clients/DownloadButtons.tsx

```diff
@@ -125,6 +120,8 @@ interface IDownloadButtons {
   desktopAppUrl: string | undefined
 }
 
+const chains = ['Ethereum','Gnosis Chain','Polygon','BNB Smart Chain','Energy Web Chain','Arbitrum','Aurora','Avalanche','Optimism','Goerli','Rinkeby','Volta']
```

Comment:
Maybe we could load these from the gateway? What do you think?

Reply:
In #1.

Reply:
I added it to the requirements.